src/renderer/components/+preferences/preferences.tsx

```diff
@@ -173,6 +173,26 @@ export class Preferences extends React.Component {
             })}
           </div>
 
+          <h2><Trans>Auto start-up</Trans></h2>
+          <Checkbox
+            label={<Trans>Open on start-up</Trans>}
+            value={preferences.openAtLogin}
+            onChange={v => preferences.openAtLogin = v}
+          />
+          <small className="hint">
+            <Trans>Opens Lens app on operation system login</Trans>
+          </small>
+
+          <h2><Trans>Tray icon</Trans></h2>
```

Comment:
No need for this preference, tray should be there always.

Reply:
Do you mean no way to disable tray for end user (as preference option)?

Reply:
Yes.